content/22.vaccines-trad.md

```diff
@@ -458,6 +459,22 @@ In general, deciding on the prioritization and allocation of the COVID-19 vaccin
 Various frameworks, models, and methods have been proposed to tackle these issues with many countries, regions, or U.S. states devising their own distribution and administration plans [@doi:10.1126/science.abe6959; @doi:10.1038/s41598-020-78447-3; @doi:10.1016/j.vaccine.2020.12.059; @doi:10.1136/medethics-2020-107036; @doi:10.1101/2021.01.12.21249694].
 The majority of the distribution plans prioritized offering vaccines to key workers such as health care workers and those who are clinically vulnerable, such as the elderly, the immunocompromised, and individuals with comorbidities, before targeting the rest of the population, who are less likely to experience severe outcomes from COVID-19 [@url:https://ourworldindata.org/covid-vaccination-policy].
 The availability of vaccines developed in a variety of countries using a variety of platforms is likely to work in favor of worldwide vaccine access.
+Hence why the development of Corbevax, a COVID-19 protein subunit vaccine developed by Texas Children’s Hospital and Baylor College of Medicine is an important initiaitve as it is patent free and so its manufactuing specifications can be shared globally. 
+Corbevax can be produced at low-cost using existing techonology that is now licensed to Biological E. Limited (BioE) for production and is approved for distribution in India and Botswana [@url:https://theconversation.com/corbevax-a-new-patent-free-covid-19-vaccine-could-be-a-pandemic-game-changer-globally-174672; @url:https://covid19.trackvaccines.org/vaccines/54/].
+
+Logistical challenges and geographical barriers also dictate the availability of certain vaccines. 
+Many countries have had poor availability of ultra low temperature freezers leading to challenges of distribution for vaccines requiring cold storage [@doi:10.1093/inthealth/ihab010; @doi:10.1208/s12249-021-01974-3; @url:https://www.bloomberg.com/news/articles/2021-01-03/pfizer-to-supply-african-health-care-workers-with-vaccines?leadSource=uverify%20wall]. 
```

Comment:
```suggestion
Many countries have had poor availability of ultra-low temperature freezers, leading to challenges of distribution for vaccines such as mRNA vaccines that require storage at very low temperatures [@doi:10.1093/inthealth/ihab010; @doi:10.1208/s12249-021-01974-3; @url:https://www.bloomberg.com/news/articles/2021-01-03/pfizer-to-supply-african-health-care-workers-with-vaccines]. 
```
I think some of them just require normal refrigeration and that is an improvement, right? I wasn't sure if "cold storage" was quite the right term since normal-refrigerator-able vaccines are being designed to try to work around this problem

Reply:
The most of the more traditional vaccines do only require normal refrigeration. I think the problem was basically the mRNA ones. Although in hotter nations, the traditional vaccines generally needed 2-8c storage, which is still a challenge in its own right when dealing with bulk amounts being transported, but otherwise fine if it is only a few doses to share.